code/offline-transactions/durable-nonce/use-nonce-account/main.en.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  clusterApiUrl,
+  Connection,
+  PublicKey,
+  Keypair,
+  Transaction,
+  SystemProgram,
+  NonceAccount,
+} from "@solana/web3.js";
+import * as bs58 from "bs58";
+
+(async () => {
+  // connection
+  const connection = new Connection(clusterApiUrl("devnet"), "confirmed");
+
+  // 5YNmS1R9nNSCDzb5a7mMJ1dwK9uHeAAF4CmPEwKgVWr8
+  const feePayer = Keypair.fromSecretKey(
```

Comment:
Could we generate the keypair and airdrop to it? Should be ok to have longer examples as we wait for #25

Reply:
updated at [a88b46c5](https://github.com/solana-dev-adv/solana-cookbook/pull/21/commits/a88b46c59dfe980b347d354a797ed17a5a92ea6b)

Reply:
Could you make Alice also random address? Best to not get from a secret key

Reply:
but in the `use-nonce-account` example. there must an account from private key unless I create a new nonce account in the last example like a full long code.

Reply:
just curious why don't use an account from a private key.
I think it is a good way to persist state and make code more simple.

Reply:
Enough people using it might make it difficult to keep the account funded on the devnet. Also, someone malicious may setup something to always remove the SOL from the devnet account. Just trying to avoid those scenarios

Reply:
got it.

Reply:
BTW make some update [here](https://github.com/solana-dev-adv/solana-cookbook/pull/21/commits/c22b9e8613579f6b0fe1404e77a4a30aef0b0562)